MapboxVision/src/main/java/com/mapbox/vision/sync/syncmanager/TelemetrySyncManager.kt

```diff
@@ -81,7 +81,8 @@ class TelemetrySyncManager(
 
         val dirFile = File(path)
 
-        removeTelemetryOverQuota(dirFile.parentFile)
+        // up to Telemetry dir
```

Comment:
Could you pls make this comment clearer?

Reply:
@yunikkk done